neonvm/controllers/functests/vm_controller_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package controllers
+package functests
```

Comment:
from DMs: if the reason to move this into a separate package is just so that it can be individually tested, that can also be accomplished with `go test <file>` -- so I'd prefer leaving this as-is unless there's other reasons as well

Reply:
I still don't quite get what you mean, but if you get value from having them separate, then fair enough, let's go with it :)

Reply:
I meant there is `vm_controller_test.go` and `vm_qmp_test.go`, so by `go test ./neonvm/controllers` we can run them both.

Also it is "aesthetically" looks better IMO that lightweight unit tests are in the same folder, but heavier functional tests are in a different folder.